math_keyboard/lib/src/foundation/tex2math.dart

```diff
@@ -290,7 +290,11 @@ class TeXParser {
         case '-':
           right = result.removeLast();
           left = result.removeLast();
-          result.add(left - right);
+          if (left.toString() == '0') {
+            result.add(-right);
+          } else {
+            result.add(left - right);
+          }
```

Comment:
😏 

Although I am wondering, does this simplify the expression during conversion? And if yes, why do we not leave it expanded. I did not check the logic here - it just looks like this would convert `4 - 3` to `1` instead of leaving it as `"4 - 3"`.

Reply:
@creativecreatorormaybenot No it does not, this adds an `Expression` not a number🙂

Reply:
@edhom Oh, right. I forgot 🙈 The operator overloads :)